docassemble/AssemblyLine/data/questions/ql_baseline.yml

```diff
@@ -148,10 +148,10 @@ code: |
     petitioners = other_parties
 ---
 depends on:
-  - user_ask_role
```

Comment:
`depends on` doesn't trigger the question, but it forces an update if the variable changes. i think we at least need to keep this trigger in place

Reply:
The variable used in this change is now `user_role` though, so we'd want it to force an update if `user_role` changes, not if ` user_ask_role` changes. I'll make sure that the `depends on` is correct in the block that sets `user_role` though.